Material.Demo/Pages/SlidersDemo.axaml

```diff
@@ -3,131 +3,257 @@
-                                Classes="Modern Ticks"
-                                TickFrequency="1" />
+                                Classes=" Ticks"
+                                TickFrequency="10" />
                     </showMeTheXaml:XamlDisplay>
                     <TextBlock />
                 </StackPanel>
             </StackPanel>
             <StackPanel>
-                <TextBlock Classes="Headline6 Subheadline2" Text="Sliders with bubble (Discrete)" />
-                <StackPanel Classes="Content" Orientation="Horizontal">
-                    <showMeTheXaml:XamlDisplay UniqueId="BubbleSlider0">
+                <TextBlock Classes="Headline6 Subheadline2" Text="Discrete slider" />
+                <StackPanel Classes="Content" Orientation="Horizontal" >
+                    <showMeTheXaml:XamlDisplay UniqueId="BubbleSlider0" >
                         <Slider IsSnapToTickEnabled="True"
-                                Classes="Discrete"
+                                Classes="material-v1 Discrete"
                                 TickFrequency="1" />
                     </showMeTheXaml:XamlDisplay>
                     <TextBlock />
                 </StackPanel>
-                <StackPanel Classes="Content" Orientation="Horizontal">
-                    <showMeTheXaml:XamlDisplay UniqueId="BubbleSlider1">
+                <StackPanel Classes="Content" Orientation="Horizontal" >
+                    <showMeTheXaml:XamlDisplay UniqueId="BubbleSlider1" >
                         <Slider IsSnapToTickEnabled="True"
-                                Classes="Modern Discrete"
+                                Classes=" Discrete"
                                 TickFrequency="10" />
                     </showMeTheXaml:XamlDisplay>
                     <TextBlock />
                 </StackPanel>
-                <StackPanel Classes="Content" Orientation="Horizontal">
-                    <showMeTheXaml:XamlDisplay UniqueId="BubbleSlider2">
+                <StackPanel Classes="Content" Orientation="Horizontal" >
+                    <showMeTheXaml:XamlDisplay UniqueId="BubbleSlider2" >
                         <Slider IsSnapToTickEnabled="True"
-                                Classes="Modern Discrete Accent"
+                                Classes=" Discrete Accent"
                                 TickFrequency="10" />
                     </showMeTheXaml:XamlDisplay>
                     <TextBlock />
                 </StackPanel>
-                <StackPanel Classes="Content" Orientation="Horizontal">
-                    <showMeTheXaml:XamlDisplay UniqueId="BubbleSlider3">
+                <StackPanel Classes="Content" Orientation="Horizontal" >
+                    <showMeTheXaml:XamlDisplay UniqueId="BubbleSlider3" >
                         <Slider IsSnapToTickEnabled="True"
-                                Classes="Modern Discrete Accent Ticks"
+                                Classes=" Discrete Accent Ticks"
```

Comment:
you can give some idea for new classes naming
but if you asking why I dont change their classes in demo, because I want do it as the last commit of this PR
plus I want keep most old classes naming until next major update to give others prepare to update.

Reply:
Actually 3.0 - is a major update already, isn't it?